src/views/site/_releaseRequestRow.php

```diff
@@ -130,6 +130,26 @@
         ),
         'format' => 'html',
     ),
+    array(
+        'attribute' => 'builds.worker.worker_name',
+        'value' => function (ReleaseRequest $releaseRequest) {
+            $buildList = $releaseRequest->builds;
+
+            $workerName = '';
+            if ( !empty($buildList[0]) && $buildList[0]->worker ) {
+                $worker = $buildList[0] ? $buildList[0]->worker : \whotrades\rds\models\Worker::instance();
```

Comment:
Зачем лишняя проверка? Мы уже в условии знаем, что $worker есть.
Плюс, по кодстайлу после "(" и до ")" мы не ставим пробелы

```suggestion
            if (!empty($buildList[0]) && ($worker = $buildList[0]->worker)) {
```

Reply:
Проверку с if добавил потом, внутреннюю при присвоении просто забыл убрать